src/mmw/mmw/settings/layer_settings.py

```diff
@@ -202,7 +202,7 @@
             'code': 'protected-lands-30m',
             'display': 'Protected Lands',
             'short_display': 'Protected Lands',
-            'css_class_prefix': 'protected-lands-30m',
+            'css_class_prefix': 'protected-lands-30m ',
```

Comment:
Just checking to make sure this space was intentional. I assume it's changed to match the CSS rule changes?

Reply:
Yes. This space allows us to have `protected-lands-30m` as a separate class.